backend/app/jobs/translations/translate_project_job.rb

```diff
@@ -0,0 +1,16 @@
+module Translations
+  class TranslateProjectJob < ApplicationJob
+    queue_as :default
+
+    rescue_from(ActiveRecord::RecordNotFound) do |exception|
+      Rails.logger.error(exception)
+      Rails.logger.error(Rails.backtrace_cleaner.clean(exception.backtrace).join("\n"))
+      # TODO: handle exception tracking
+    end
+
+    def perform(project_id)
+      project = Project.find(project_id)
+      Translations::TranslateProject.new(project).call
```

Comment:
no big deal, but I believe this will be always called via this job, so maybe logic could be moved here and no need to have special service for it

Reply:
I prefer skinny jobs, as it is then easier to use the same service in the console or rake task or somewhere else entirely, and also - writing specs for a service is easier than writing specs for a job I think :)